Specs/DateTime/French/DateTimeModel.json

```diff
@@ -3411,5 +3411,121 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Je retournerai sept heures dix",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java",
+    "Results": [
+      {
+        "Text": "sept heures dix",
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T07:10",
+              "type": "time",
+              "value": "07:10:00"
+            },
+            {
+              "timex": "T19:10",
+              "type": "time",
+              "value": "19:10:00"
+            }
+          ]
+        },
+        "Start": 15,
+        "End": 29
+      }
+    ]
+  },
+  {
+    "Input": "Je retournerai sept heures et dix",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java",
+    "Results": [
+      {
+        "Text": "sept heures et dix",
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T07:10",
+              "type": "time",
+              "value": "07:10:00"
+            },
+            {
+              "timex": "T19:10",
+              "type": "time",
+              "value": "19:10:00"
+            }
+          ]
+        },
+        "Start": 15,
+        "End": 32
+      }
+    ]
+  },
+  {
+    "Input": "Je retournerai sept heures moins dix",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java",
+    "Results": [
+      {
+        "Text": "sept heures moins dix",
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T06:50",
+              "type": "time",
+              "value": "06:50:00"
+            },
+            {
+              "timex": "T18:50",
+              "type": "time",
+              "value": "18:50:00"
+            }
+          ]
+        },
+        "Start": 15,
+        "End": 35
+      }
+    ]
+  },
+  {
+    "Input": "le soleil se couchant demain à sept heures et vingt-sept minutes",
```

Comment:
Can you add a corresponding case in English?
It's important to always add to English specs as they are used as the target for coverage across languages and used as localization source when implementing a new one.

Reply:
yes, I added the corresponding English case already.

Reply:
I meant a case that ends in "minutes". But, yeah, it's not very common...

Reply:
ah I see, ok I will add it as well.

Reply:
done